store/storetest/channel_store.go

```diff
@@ -978,7 +978,7 @@ func testChannelDeleteMemberStore(t *testing.T, ss store.Store) {
 		t.Fatal("should have saved 2 members")
 	}
 
-	store.Must(ss.Channel().PermanentDeleteMembersByUser(o2.UserId))
+	ss.Channel().PermanentDeleteMembersByUser(o2.UserId)
```

Comment:
To be consistent with the previous behavior, I miss handling the error here, something like store the error response in an `err` variable and a the line `require.Nil(t, err)`

Reply:
Done.